tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -1900,6 +1905,24 @@ bool DeclResultIdMapper::createStageVars(
               astContext.getExtVectorType(srcVecElemType, 2), *value, *value,
               {0, 1}, thisSemantic.loc);
       }
+      // Special handling of SV_ShadingRate, which is a bitpacked enum value,
+      // but SPIR-V's FragSizeEXT uses a uint2. We build the enum value from
```

Comment:
IIUC the comment suggests FragSizeEXT uses a uint2. But the code is using int2. Either the comment or the code should be updated.

Reply:
Indeed, this comment is wrong. I will fix, thanks.